config/apiserver/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hiveapi
+  namespace: hive
+  labels:
+    api: hiveapi         
+    apiserver: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      api: hiveapi
+      apiserver: "true"
+  template:
+    metadata:
+      labels:
+        api: hiveapi
+        apiserver: "true"
+    spec:
+      containers:
+      # By default we will use the latest CI images published from hive master:
+      - image: registry.svc.ci.openshift.org/openshift/hive-v4.0:hive
```

Comment:
Shouldn't this be registry.svc.ci.openshift.org/openshift/hivev1:hive? I'm worried something's hiding issues in your test flow because I think this is the v1alpha1 image that shouldn't have the binary we're trying to run here.

Reply:
i've been testing with my own custom containers, since that's where all the new hive operator code is